makelib/variables.mk.in

```diff
@@ -18,7 +18,7 @@ REPO_PATH := $(ORG_PATH)/rkt
 # [STAGE1] build settings
 
 # selinux tags for rkt and functional tests
-RKT_TAGS := -tags selinux
+RKT_TAGS := -tags selinux @TPM_TAGS@
```

Comment:
To specify multiple tags, you must put them inside quotes. So `-tags 'selinux @TPM_TAGS@'` should ok.